### ISSUE TITLE
hi! Could you also make it pi5 compatible too?

### ISSUE THREAD

**Issue by despauterio** [ChristopheJacquet/PiFmRds#195]
hi! Thanks.
Could you eventually also make it pi5 compatible too?
I could try and test it if it helps obviously :)
seems to run fine on Pi4b, and thanks for that!

**Comment by ChristopheJacquet** [comment]
I don't have a RPi 5 to test unfortunately... Just checking: when you run `make`, do the `gcc` lines have `-DRASPI=4`, meaning it's trying to compile with the same settings as on a RPi 4?

**Comment by despauterio** [comment]
true!
"gcc -Wall -std=gnu99 -c -g -march=armv8-a -O2 -pipe -fstack-protector-strong -fno-plt -ffast-math -DRASPI=4 mailbox.c"
will try with your new tag!!

**Comment by notluca** [comment]
> I don't have a RPi 5 to test unfortunately... Just checking: when you run `make`, do the `gcc` lines have `-DRASPI=4`, meaning it's trying to compile with the same settings as on a RPi 4?

That is correct, I figurerd out RPI5 has a new chip got GPIO,IS2 etc.. It's called "RP1". If you need a tester for the RPI5, you can contact me on @notlucadev on discord.
